### PR TITLE
Making few important updates to secret handler component

### PR DESCRIPTION
**Pull request by devsatishm**

### CODE REVIEW

File: api-template-java/src/main/java/com/slf/services/Handler.java [hunk 1]
```diff
@@ -49,9 +42,20 @@
 	// @Override
 	public Response handleRequest(Request input, Context context) {
 
+		// Initialize the environment specific variables. Each environment has different properties files to store env specific data.
+		// For example, for development environment specific details, add configurations as key,value pairs in 'dev.properties'
```
Comment (suryajak) [inline api-template-java/src/main/java/com/slf/services/Handler.java]: What if I don't use any configuration value? This code should be available as a template but commented out.

File: api-template-nodejs/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // =========================================================================
-// Copyright � 2017 T-Mobile USA, Inc.
-// 
+// Copyright � 2017 T-Mobile USA, Inc.
```
Comment (suryajak) [inline api-template-nodejs/index.js]: Might want to check why this character isn't showing up on your machine.

File: jenkins-build-pack-website/Jenkinsfile [hunk 3]
```diff
@@ -273,16 +271,16 @@ node ()  {
 		}
 	  }
 	  else {
-	  
-	  
+
+
 		stage ('Deployment to Deployement Env'){
```
Comment (suryajak) [inline jenkins-build-pack-website/Jenkinsfile]: typo?

File: lambda-template-nodejs/index.js [hunk 4]
```diff
@@ -21,28 +21,17 @@
 **/
 
 const config = require('./components/config.js'); //Import the environment data.
-const secretHandlerModule = require("./components/secret-handler.js"); //Import the secret-handler module.
 const logger = require("./components/logger.js"); //Import the logging module.
+const responseObj = require("./components/response.js"); //Import the response module.
```
Comment (suryajak) [inline lambda-template-nodejs/index.js]: Include error-handler that you have added as a component and also add a (commented out) code snippet as to how to use it

File: platform-services-handler/index.js [hunk 5]
```diff
@@ -61,9 +61,9 @@ module.exports.handler = (event, context, cb) => {
 		eventVersion: '1.0',
 		eventID: 'shardId-000000000000:49574219880753003597816065353678073460941546253285588994',
 		eventName: 'aws:kinesis:record',
-		invokeIdentityArn: 'arn:aws:iam::302890901340:role/lambda_basic_execution',
+		invokeIdentityArn: 'arn:aws:iam::xxx:role/lambda_basic_execution',
```
Comment (suryajak) [inline platform-services-handler/index.js]: use config variables to set this (as we do for our other services)

File: platform_login/package.json [hunk 6]
```diff
@@ -1,34 +1,34 @@
...
+  "version": "0.0.1",
+  "description": "Authentication and Token Service",
+  "main": "index.js",
+  "private": true,
+  "author": [],
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "chai": "^3.5.0",
```
Comment (suryajak) [inline platform_login/package.json]: Remove chai, mocha, aws-sdk

File: platform_login/package.json [hunk 7]
```diff
@@ -1,34 +1,34 @@
...
+    "deasync": "*",
+    "async": "*",
+    "jsonwebtoken": "*",
+    "request": "*",
+    "aws-sdk": "*",
+    "moment": "*",
+    "amazon-cognito-identity-js": "^1.19.0"
+  },
+  "devDependencies": {
+    "activedirectory": "*",
```
Comment (suryajak) [inline platform_login/package.json]: Remove the ones that are in dependencies. I also think activedirectory isn't required here (haven't looked at the code)

File: platform_logout/package.json [hunk 8]
```diff
@@ -11,10 +11,10 @@
   "dependencies": {
     "request": "*",
     "deasync": "*",
-	"async":"*",
-	"jsonwebtoken": "*",
-	"aws-sdk":"*",
-	"amazon-cognito-identity-js": "^1.19.0"
+    "async": "*",
+    "jsonwebtoken": "*",
+    "aws-sdk": "*",
```
Comment (suryajak) [inline platform_logout/package.json]: remove aws-sdk

Comment (devsatishm) [reply platform_logout/package.json]: Done.

File: platform_login/package.json (hunk 7 above)
Comment (devsatishm) [reply platform_login/package.json]: Done.

File: platform_login/package.json (hunk 6 above)
Comment (devsatishm) [reply platform_login/package.json]: Done.

File: platform-services-handler/index.js (hunk 5 above)
Comment (devsatishm) [reply platform-services-handler/index.js]: This is just an example event for reference with random account id. Just getting rid of all the account numbers from the code.

File: lambda-template-nodejs/index.js (hunk 4 above)
Comment (devsatishm) [reply lambda-template-nodejs/index.js]: Done.

File: jenkins-build-pack-website/Jenkinsfile (hunk 3 above)
Comment (devsatishm) [reply jenkins-build-pack-website/Jenkinsfile]: Yep, changed.

File: api-template-java/src/main/java/com/slf/services/Handler.java (hunk 1 above)
Comment (devsatishm) [reply api-template-java/src/main/java/com/slf/services/Handler.java]: Done.

**Pull request merged**